index.js

```diff
@@ -0,0 +1,95 @@
+const fs = require("fs");
+const { Readable } = require("node:stream");
+
+class APIClient
+{
+  static async login(url, username, password) {
+    const loginUrl = url + "/api/auth/jwt/login";
+
+    const form = new FormData();
+    form.append("username", username);
+    form.append("password", password);
+
+    const res = await fetch(loginUrl, {method: "POST", body: form});
+    const auth = await res.json();
+    const {token_type, access_token} = auth;
+
+    const authToken = token_type + " " + access_token;
+    return new APIClient(url, authToken);
+  }
+
+  constructor(url, auth) {
+    this.url = url;
+    this.auth = auth;
+  }
+
+  async fetchAPI(endpoint, method="GET", body=null) {
+    const headers = {"Authorization": this.auth, "Content-Type": "application/json"};
+    const resp = await fetch(this.url + endpoint, {headers, method, body, duplex: "half"});
+    return await resp.json();
+  }
+
+  async getOrg(name="") {
+    const json = await this.fetchAPI("/api/users/me-with-orgs");
+    const { orgs } = json;
+    if (!orgs || !orgs.length) {
+      return null;
+    }
+    if (!name) {
+      return orgs[0].id;
+    }
+    for (const org of orgs) {
+      if (org.name === name) {
+        return org.id;
+      }
+    }
+    return orgs[0].id;
+  }
+
+	async getCrawlerWorkflows(org) {
+		const res = await this.fetchAPI(`/api/${org}/crawlconfigs`)
+		return res;
+	}
+
+	async getCrawlerWorkflow(org, cid="") {
+		const res = await this.fetchAPI(`/api/${org}/crawlconfigs/${cid}`)
+		return res;
```

Comment:
Same as above for these - first line is missing a semi-colon, but also can just `return await ...`